providers/universal-provider/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "@walletconnect/universal-provider",
+  "description": "Universal Provider for WalletConnect Protocol",
+  "version": "2.0.0-rc.2",
+  "author": "WalletConnect, Inc. <walletconnect.com>",
+  "homepage": "https://github.com/walletconnect/walletconnect-monorepo/",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/walletconnect/walletconnect-monorepo",
+    "directory": "packages/universal-provider"
+  },
+  "license": "Apache-2.0",
+  "main": "dist/index.cjs.js",
+  "module": "dist/index.es.js",
+  "unpkg": "dist/index.umd.js",
+  "types": "dist/types/index.d.ts",
+  "files": [
+    "dist"
+  ],
+  "keywords": [
+    "wallet",
+    "walletconnect"
+  ],
+  "scripts": {
+    "clean": "rm -rf dist",
+    "build:pre": "npm run clean",
+    "build:types": "tsc",
+    "build:source": "rollup --config rollup.config.js",
+    "build": "npm run build:pre; npm run build:source; npm run build:types",
+    "test:pre": "rm -rf ./test/test.db",
```

Comment:
Think this can be removed since it's a side effect for running tests in sign-client specificllay